pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/InstanceUtils.java

```diff
@@ -103,4 +106,21 @@
         }
         return SINK;
     }
+
+    public static Map<String, String> getProperties(Utils.ComponentType componentType, String fullyQualifiedInstanceId) {
+        Map<String, String> properties = new HashMap<>();
+        switch (componentType) {
+            case FUNCTION:
+                properties.put("application", "pulsar-function");
+                break;
+            case SOURCE:
+                properties.put("application", "pulsar-source");
+                break;
+            case SINK:
+                properties.put("application", "pulsar-sink");
+                break;
+        }
+        properties.put("id", fullyQualifiedInstanceId);
```

Comment:
should we have a name field too?

Reply:
we could have separate fields for tenant, namespace, and name.  What do you think?

Reply:
I think just a field with fully qualified name is fine
maybe:
application = pulsar-function|pulsar-sink|pulsar-source
id = tenant/namespace/name
instanceId = instanceId

what do you think?

Reply:
@cckellogg sounds good. I have updated the PR